root/etc/confd/templates/basic-security.groovy.tmpl

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+import jenkins.model.*
+import hudson.security.*
+
+def instance = Jenkins.getInstance()
+
+println "--> creating local user 'admin'"
+
+def hudsonRealm = new HudsonPrivateSecurityRealm(false)
+hudsonRealm.createAccount('admin','{{getenv "JENKINS_ADMIN_PASS"}}')
```

Comment:
So if this env var is not set, this just fails gracefully and no login is required, or there is a login but you don't enter a password?